src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -26,31 +25,30 @@ internal class CookieAuthenticationHandler : AuthenticationHandler<CookieAuthent
         private DateTimeOffset? _renewIssuedUtc;
         private DateTimeOffset? _renewExpiresUtc;
         private string _sessionKey;
-        private Task<AuthenticationTicket> _cookieTicketTask;
+        private Task<AuthenticateResult> _readCookieTask;
 
-        private Task<AuthenticationTicket> EnsureCookieTicket()
+        private Task<AuthenticateResult> EnsureCookieTicket()
         {
             // We only need to read the ticket once
-            if (_cookieTicketTask == null)
+            if (_readCookieTask == null)
             {
-                _cookieTicketTask = ReadCookieTicket();
+                _readCookieTask = ReadCookieTicket();
             }
-            return _cookieTicketTask;
+            return _readCookieTask;
         }
 
-        private async Task<AuthenticationTicket> ReadCookieTicket()
+        private async Task<AuthenticateResult> ReadCookieTicket()
         {
             var cookie = Options.CookieManager.GetRequestCookie(Context, Options.CookieName);
             if (string.IsNullOrEmpty(cookie))
             {
-                return null;
+                return AuthenticateResult.Fail($"'{Options.CookieName}' cookie not found.");
```

Comment:
We don't actually log cookie failures at all, that behavior only is part of Remote based Auth Handlers.  The 'failures' in cookie are on the result that no one looks at, so they are silently ignored right now.  You would be able to see them if you called Authenticate using the context overload that returned the Result.